app/styles/headers/_fifth.scss

```diff
@@ -113,9 +113,10 @@
         transform: translateY(-50%);
 
         i {
-            position: absolute;
```

Comment:
1. How without `position:absolute` you shift right arrow ?
2. This arrows in the end html - they don't put in the right place without `position:absolute` and be after all blocks.

Reply:
Test approved - all good.
My mistake.